core/settings.py

```diff
@@ -301,10 +354,11 @@ def set_nested(self, key_tuple, value, supress_missing=False):
                 c: 	True
         """
         s = self
+        *initial_keys, final_key = key_tuple
```

Comment:
Rewrote this such that ``key_tuple`` can be any type of iterable and not just a sequence.